### PR TITLE
Style: How works section

### PR DESCRIPTION
**Pull request by varffs**
- with alt markup for mobile

### CODE REVIEW

File: src/styl/responsive/desktop.styl [hunk 1]
```diff
@@ -1,3 +1,10 @@
+// HOW IT WORKS
+#section-how
+  .how-illustration-caption
+    font-size: 12px
+    svg
+      margin-bottom: $margin-tiny
```
Comment (mikeyrayvon) [inline src/styl/responsive/desktop.styl]: this could go on the elem with class `.margin-bottom-tiny`

Comment (mikeyrayvon) [inline src/styl/responsive/desktop.styl]: this could go on the elem with class `.margin-bottom-tiny`

**Review by mikeyrayvon — COMMENTED** [review]
just consider moving that bottom margin from `.styl` to the element attribute

**Comment by varffs** [comment]
edit done @mikeyrayvon

**Pull request merged**